test/common_test/detachable_joint.cc

```diff
@@ -0,0 +1,202 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+#include <gtest/gtest.h>
+
+#include <chrono>
+
+#include <gz/common/Console.hh>
+#include <gz/plugin/Loader.hh>
+
+#include <gz/math/Helpers.hh>
+#include <gz/math/eigen3/Conversions.hh>
+
+#include "TestLibLoader.hh"
+#include "../Utils.hh"
+
+#include "gz/physics/FrameSemantics.hh"
+#include <gz/physics/FindFeatures.hh>
+#include <gz/physics/FixedJoint.hh>
+#include <gz/physics/ForwardStep.hh>
+#include <gz/physics/FreeGroup.hh>
+#include <gz/physics/FreeJoint.hh>
+#include <gz/physics/GetEntities.hh>
+#include <gz/physics/Joint.hh>
+#include <gz/physics/RemoveEntities.hh>
+#include <gz/physics/RequestEngine.hh>
+#include <gz/physics/RevoluteJoint.hh>
+#include <gz/physics/Shape.hh>
+#include <gz/physics/sdf/ConstructModel.hh>
+#include <gz/physics/sdf/ConstructWorld.hh>
+
+#include <sdf/Root.hh>
+
+template <class T>
+class DetachableJointTest:
+  public testing::Test, public gz::physics::TestLibLoader
+{
+  // Documentation inherited
+  public: void SetUp() override
+  {
+    gz::common::Console::SetVerbosity(4);
+
+    std::cerr << "DetachableJointTest::GetLibToTest() " << DetachableJointTest::GetLibToTest() << '\n';
```

Comment:
remove ?

Reply:
This is something we do in the other common tests (eg https://github.com/gazebosim/gz-physics/blob/128f74066fbad7158927e45454e6741785fbafa5/test/common_test/joint_features.cc#L56).  Is there a reason to specifically remove it here?